### PR TITLE
Tagging of Blueprints and Items on Canvas

### PR DESCRIPTION
**Pull request by dtaylor113**
Implemented Tagging of Blueprints:

image

Implemented Tagging of Items on Canvas:

image

Reverted eslint changes for Blueprint files.

I originally had this PR as two smaller ones, but ran into rebase/merge issues with the eslint changes, so had to fix everything in one PR.

### CODE REVIEW

**Comment by dtaylor113** [comment]
@serenamarie125 -ping-

**Comment by dtaylor113** [comment]
Associated JIRA story: https://patternfly.atlassian.net/browse/CFUX-90
@chriskacerguis , @serenamarie125

**Comment by AllenBW** [comment]
:metal: <3

**Comment by chriskacerguis** [comment]
@dtaylor113 is this ready to be reviewed / merged?

**Comment by serenamarie125** [comment]
LGTM from a UX perspective @chriskacerguis @dtaylor113

**Comment by chriskacerguis** [comment]
@serenamarie125 great, waiting to hear from Dave if this is ready for merge.

File: client/app/components/blueprint-details-modal/order-list.directive.js [hunk 1]
```diff
@@ -38,20 +38,18 @@
...
+      controller: OrderListController
     };
 
     return directive;
   }
 
   function OrderListController($scope, $filter) {
-    var vm = this;
-    vm.listDisabled = false;
+    $scope.listDisabled = false;
```
Comment (chriskacerguis) [inline client/app/components/blueprint-details-modal/order-list.directive.js]: So, what are we doing with $scope?  @AllenBW pulled it out, and you are putting it back in (just wondering what's the long term plan)?

Comment (dtaylor113) [inline client/app/components/blueprint-details-modal/order-list.directive.js]: @AllenBW only changed the javascript and not the corresponding HTML, and none of the blueprint functionality was tested before the PR was merged.   Long term I think one blueprint component at a time should be converted and tested.  @serenamarie125 should I create corresponding JIRA stories for converting components to pass eslint?  -thanks

**Comment by chriskacerguis** [comment]
👍

File: client/app/components/blueprint-details-modal/order-list.directive.js (hunk 1 above)
Comment (chriskacerguis) [inline client/app/components/blueprint-details-modal/order-list.directive.js]: @dtaylor113 @serenamarie125 FWIW - Yes, we need to start getting things to pass eslint.

Comment (dtaylor113) [inline client/app/components/blueprint-details-modal/order-list.directive.js]: Hi, @chriskacerguis , just IM'ing with @serenamarie125 , she is creating eslint JIRA stories to cover this work.

**Pull request merged**